docs/application/dotnet/tutorials/library/library-list.md

```diff
@@ -0,0 +1,57 @@
+
+# Third Party Libraries for Tizen .NET Application
+
+In this list, you can find the libraries organized by purpose.
+
+- [Hardware](#hardware)
+- [Messaging](#messaging)
+- [Multimedia](#multimedia)
+- [Security](#security)
+- [System](#system)
+- [User interface](#user-interface)
+
+## Hardware
+
+*Libraries for programming with hardware.*
+
+* [Barcode Scanner](https://github.com/Redth/ZXing.Net.Mobile) - Scan and create barcodes with ZXing.NET.Mobile.
+* [Battery Status](https://github.com/jamesmontemagno/BatteryPlugin) - Gather battery level, charging status, and type.
+* [Connectivity](https://github.com/jamesmontemagno/ConnectivityPlugin) - Get network connectivity info such as type and if connection is available.
+* [Device Info](https://github.com/jamesmontemagno/DeviceInfoPlugin) - Properties about device such as OS, Model, and Id.
+* [Device Motion](https://github.com/rdelrosario/xamarin-plugins/tree/master/DeviceMotion) - Provides access to Accelerometer, Gyroscope, Magnetometer, and Compass.
+
+## Messaging
+
+*Libraries for manipulating message.*
+
+* [Messaging](https://github.com/cjlotz/Xamarin.Plugins) - Make phone call, send sms, and send e-mail
+
+## Multimedia
+
+*Libraries for manipulating multimedia.*
+
+* [FFImageLoading](https://github.com/molinch/FFImageLoading) - Image loading with caching, placeholders, transformations and more
+* [SimpleAudioPlayer](https://github.com/adrianstevens/Xamarin-Plugins/tree/master/SimpleAudioPlayer) - SimpleAudioPlayer plays local files and audio data as a stream. This allows you to store audio data in a portable class library and play it on all supported platforms.
+* [SkiaSharp](https://github.com/mono/SkiaSharp) - The Skia 2D Graphics library from Google exposed to .NET languages and runtimes across the board
+* [Media](https://github.com/jamesmontemagno/MediaPlugin) - Take or pick photos and videos.
+* [Media Manager](https://github.com/martijn00/XamarinMediaManager) - Playback for Audio.
+
+## Security
+
+*Libraries for manipulating secure storage.*
+
+* [Secure Storage](https://github.com/sameerkapps/SecureStorage) - Provides secure storage for key value pairs Data
+
+## System
+
+*Libraries for manipulating system.*
+
+* [Permissions](https://github.com/jamesmontemagno/PermissionsPlugin) - Easily check and request runtime permissions.
+* [Settings](https://github.com/jamesmontemagno/SettingsPlugin) - Simple & Consistent cross platform settings API.
+
+## User interface
+
+*Libraries for working with graphical user interface applications.*
+
+* [Share](https://github.com/jguertl/SharePlugin) - Easily share text, links, or open a browser.
+* [User Dialogs](https://github.com/aritchie/userdialogs) - Message box style dialogs
```

Comment:
Can we rephrase as:
Dialog boxes consisting of messages.
Please share your opinion before applying the changes.

Reply:
This library is for a dialog with message box style.

new sentence:
Calls for standard user dialogs from a core .net standard library, Actionsheets, alerts, confirmations, loading, login, progress, prompt, toast.

Reply:
Suggestion:
This library is for standard user dialog from core .net standard libraries such as actionsheets, alerts, confirmations, loading, login, progress, prompt, and toast.
or
This library is for standard user dialog from core .net standard library, actionsheets, alerts, confirmations, loading, login, progress, prompt, and toast.

Please verify and share your opinion.
Please correct me if I am wrong.

Reply:
done